src/main/java/org/graylog/integrations/aws/AWSLogMessage.java

```diff
@@ -0,0 +1,45 @@
+package org.graylog.integrations.aws;
+
+/**
+ * A helper class that supports the ability to detect the type of AWS log message.
+ */
+public class AWSLogMessage {
+
+    private String logMessage;
+
+    public AWSLogMessage(String logMessage) {
+        this.logMessage = logMessage;
+    }
+
+    /**
+     * Detects the type of log message.
+     *
+     * @return A {@code Type} indicating the which kind of log message has been detected.
+     */
+    public AWSMessageType detectLogMessageType() {
+
+        if (isFlowLog()) {
+            return AWSMessageType.KINESIS_FLOW_LOGS;
+        }
+
+        return AWSMessageType.KINESIS_RAW;
+    }
+
+    /**
+     * Flow logs are space-delimited messages. See https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html
+     * <p>
+     * Sample: 2 123456789010 eni-abc123de 172.31.16.139 172.31.16.21 20641 22 6 20 4249 1418530010 1418530070 ACCEPT OK
+     * <p>
+     * Match a message with exactly 13 spaces and either the word ACCEPT or REJECT.
+     * Use simple if checks instead of regex to keep this simple. Performance should not be a concern, since
+     * this is only called once during the healthcheck.
+     *
+     * @return true if message is a flow log.
+     */
+    private boolean isFlowLog() {
+        boolean hasAction = logMessage.contains("ACCEPT") || logMessage.contains("REJECT");
```

Comment:
@bernd Good point on checking if null to avoid NPE. Would it be better to silently return `AWSMessageType.KINESIS_RAW` if the message is null? The reason is, we still want the Health Check to continue and return a result even if the message is null. The UI would then display a result to the user indicating that a blank message was found (probably unlikely that a user would write such a message).